README.md

```diff
@@ -10,11 +10,16 @@ Go clients for talking to a [kubernetes](http://kubernetes.io/) cluster.
 * The `transport` package is used to set up auth and start a connection.
 * The `tools/cache` package is useful for writing controllers.
 
-### Releases
+### Versions
```

Comment:
> I think you can redirect go get client-go/1.5 to the 1.5 branch

I take this back. I misread the "go-source" tag. `go get --help` says 
```
The most important rule is that if the local installation is running version 
"go1", get searches for a branch or tag named "go1". If no such version exists it
retrieves the most recent version of the package
```
So it seems we cannot redirect `go get` to checkout a branch.